postgres/getting-started/create-pg-cluster.html.md.erb

```diff
@@ -17,46 +17,60 @@ fly postgres create
 ```
 ```output
 ? Choose an app name (leave blank to generate one): pg-test
-automatically selected personal organization: TestOrg
-? Select regions:  [Use arrows to move, type to filter]
-> Amsterdam, Netherlands (ams)
-  Paris, France (cdg)
-  Denver, Colorado (US) (den)
-  ...
-  Sydney, Australia (syd)
-  Montreal, Canada (yul)
-  Toronto, Canada (yyz)
-? Select regions: Miami, Florida (US) (mia)
+? Select Organization: TestOrg (personal)
+Some regions require a paid plan (fra, maa).
+See https://fly.io/plans to set up a plan.
+
+? Select region: Toronto, Canada (yyz)
 ```
 
 During this process, you get to choose from several preset resource configurations for the app:
 
 ```
 ? Select configuration:  [Use arrows to move, type to filter]
 > Development - Single node, 1x shared CPU, 256MB RAM, 1GB disk
-  Production - Highly available, 2x shared CPUs, 4GB RAM, 40GB disk
-  Production - Highly available, 4x shared CPUs, 8GB RAM, 80GB disk
+  Production (High Availability) - 3 nodes, 2x shared CPUs, 4GB RAM, 40GB disk
+  Production (High Availability) - 3 nodes, 4x shared CPUs, 8GB RAM, 80GB disk
   Specify custom configuration
 ```
 
-The "Production" options give you a two-node cluster in a leader-replica configuration. A single-node "Development" instance can readily be scaled and [expanded to more regions](/docs/postgres/advanced-guides/high-availability-and-global-replication/).
+The "Production" options give you a three-node cluster in a leader-replica configuration. A single-node "Development" instance can readily be scaled and [expanded to more regions](/docs/postgres/advanced-guides/high-availability-and-global-replication/).
+
+If you select the "Development" single-node cluster configuration, then you can choose to scale down to zero if there are no open connections after one hour.
 
 ```
-Creating postgres cluster pg-test in organization TestOrg
+? Scale single node pg to zero after one hour? (y/N)
+```
+
+<div class="callout">
+You need to configure your app to scale to zero as well, otherwise your Postgres database will never scale down. For Phoenix apps, for example, you can [shut down the app when it's idle](https://fly.io/phoenix-files/shut-down-idle-phoenix-app/). Learn more about the [scale to zero setting](/docs/postgres/getting-started/scale-to-zero/), including how to turn it off.
```

Comment:
@catflydotio - In the case of this note, "your app" is the consuming app, right? 

I saw [this section](https://fly.io/docs/apps/autostart-stop/#stop-a-machine-by-terminating-its-main-process) that you added to the autostart and stop docs, and I'm wondering if it's useful in more cases and should be it's own page that we can reference? Or a partial?

Reply:
>  In the case of this note, "your app" is the consuming app, right?

Yes, there may also be a better way to put it than "consuming app" 😄 😅 

> I saw [this section](https://fly.io/docs/apps/autostart-stop/#stop-a-machine-by-terminating-its-main-process) that you added to the autostart and stop docs

I moved that section from wherever it was before, because I figured it's mainly relevant when you're using the proxy to autostart. I'm OK with moving it if there's somewhere better.

Reply:
made some minor edits to address most of this to some degree